editor/editor_properties_array_dict.cpp

```diff
@@ -458,6 +459,10 @@ bool EditorPropertyArray::_is_drop_valid(const Dictionary &p_drag_data) const {
 		allowed_type = subtype_hint_string;
 	}
 
+	if (allowed_type == "NodePath") {
+		return true;
```

Comment:
This is wrong. It allows to drop anything in NodePath array, resulting in errors on mismatch.

Reply:
Yes that's something I realized I forgot to change yesterday but I was waiting for more feedbacks to change that at the same time.